_articles/send/send-lifespan.md

```diff
@@ -0,0 +1,209 @@
+  </li>
+  <li class="nav-item" role="presentation">
+    <a class="nav-link" id="clitab" href="#cli" role="tab" aria-controls="cli" aria-selected="false">CLI</a>
+  </li>
+</ul>
+<div class="tab-content" id="clientsContent">
+  <div class="tab-pane show active" id="webvault" role="tabpanel" aria-labelledby="wvtab">
+{% capture web_vault %}
+
+#### Disable from the Web Vault
+
+To disable a Send from the Web Vault, open the **Edit Send** view and check the **Disable this Send so that no one can access it** checkbox. You can uncheck this box at any time to re-enable access to the Send.
+
+{% image /send/send-disable-wv.png Disable from the Web Vault %}
+
+When a Send is disabled:
+
+- For recipients of the Send (i.e. anyone with the generated link), navigating to the Send link will show a screen reporting that the Send does not exist or is no longer available.
+
+- For the Sender, a {% icon fa-warning %} **Disabled** icon will appear next to the Send. The Send will remain accessible to its Sender in the Send view until the specified **Deletion Date** is reached.
+
+#### Delete from the Web Vault
+
+To delete a Send from the Web Vault, use the hover-over {% icon fa-cog %} gear dropdown to select the {% icon fa-trash %} **Delete** button:
+
+{% image /send/send-delete.png Delete from the Web Vault %}
+
+{% endcapture %}
+{{ web_vault | markdownify}}
+  </div>
+  <div class="tab-pane" id="browserextension" role="tabpanel" aria-labelledby="betab">
+{% capture browser_extension %}
+
+#### Disable from Browser Extensions
+
+To disable a Send from a Browser Extension, open the **Edit Send** view and check the **Disable this Send so that no one can access it** checkbox. You can uncheck this box at any time to re-enable access to the Send.
+
+{% image /send/send-disable-be.png Disable from a Browser Extension %}
+
+When a Send is disabled:
+
+- For recipients of the Send (i.e. anyone with the generated link), navigating to the Send link will show a screen reporting that the Send does not exist or is no longer available.
+
+- For the Sender, a {% icon fa-warning %} **Disabled** icon will appear next to the Send. The Send will remain accessible to its Sender in the Send view until the specified **Deletion Date** is reached.
+
+#### Delete from Browser Extensions
+
+To delete a Send from a Browser Extension, select the {% icon fa-trash %} **Trash** icon next to the Send you want to delete:
+
+{% image /send/send-delete-be.png Delete from a Browser Extension %}
+
+{% endcapture %}
+{{ browser_extension | markdownify}}
+  </div>
+  <div class="tab-pane" id="desktop" role="tabpanel" aria-labelledby="desktab">
+{% capture desktop_info %}
+
+#### Disable from Desktop Apps
+
+To disable a Send from a Desktop App, open the **Edit Send** view and check the **Disable this Send so that no one can access it** checkbox. You can uncheck this box at any time to re-enable access to the Send.
+
+{% image /send/send-desktop-disable.png Disable from a Desktop App%}
```

Comment:
@fschillingeriv , this is the wrong image (shows the "hide text" option vs. disable option).